centml/values.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "properties": {
+    "nodeSelector": {
+      "type": "object",
+      "form": true,
+      "title": "Node Selection",
+      "required": ["region"],
+      "properties": {
+        "region": {
+          "type": "string",
+          "enum": ["ORD1", "EWR1", "LAS1", "LGA1"],
+          "form": true,
+          "title": "Region",
+          "customComponent": {
+            "id": "region_selector",
+            "component": "RegionSelector",
+            "description": "Select your preferred region for your Jupyter instance",
+            "title": "Geographic Location"
+          }
+        }
+      }
+    },
+    "imageTag": {
+      "type": "integer",
+      "form": true,
+      "title": "Image tag",
+      "enum": [1]
```

Comment:
An image tag value should probably required.

Reply:
Also I would remove this enum since the tags are indefinite. Also as it stands the chart would not work since `1.0.0` is in the values file.

Reply:
I have set `imageTag` as required and changed the type to string to support `1.0.0`